Bottom: 3cc27df75e771112a05a2a64a3e4519de8113374
Top:    eafa85605960c59140ba08d65e7da8a08fc638fb
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-10 11:38:13 +0800

Refresh of tun-xdp-support

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 7e92192..00a3ec7 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1022,10 +1022,6 @@ static int tun_xdp_set(struct net_device *dev, struct bpf_prog *prog,
 	struct tun_struct *tun = netdev_priv(dev);
 	struct bpf_prog *old_prog;
 
-	/* We will shift the packet that can't be handled to generic
-	 * XDP layer.
-	 */
-
 	old_prog = rtnl_dereference(tun->xdp_prog);
 	rcu_assign_pointer(tun->xdp_prog, prog);
 	if (old_prog)
